src/javascript/app/base/header.js

```diff
@@ -79,9 +81,10 @@ const Header = (() => {
                     const account_title  = Client.getAccountTitle(loginid);
                     const is_real        = !Client.getAccountType(loginid); // this function only returns virtual/gaming/financial types
                     const currency       = Client.get('currency', loginid);
-                    const localized_type = localize('[_1] Account', is_real && currency ? getCurrencyDisplayCode(currency) : account_title);
+                    const data = { currency , is_real , account_title };
+                    const localized_type = getLocalizedType(data);
                     if (loginid === Client.get('loginid')) { // default account
-                        applyToAllElements('.account-type', (el) => { elementInnerHtml(el, localized_type); });
+                        applyToAllElements('.account-type', (el) => { elementInnerHtml(el, getLocalizedType({ ...data , account_title: 'Demo' })); });
```

Comment:
why are you hardcoding the account title to demo here? why not update it in the getAccountTitle function to return demo instead of virtual? we have this code `virtual  : localize('Virtual'),` that if changed to `virtual  : localize('Demo'),` should make any changes here unnecessary.

Reply:
done